src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,15 +1,108 @@
 import { Component } from 'react';
+import './NewMovie.scss';
 
-type Props = {};
-type State = {};
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
+type State = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
 
 export class NewMovie extends Component<Props, State> {
-  state: State = {};
+  state: State = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleChange = (value: string, key: string) => {
+    this.setState(state => ({
+      ...state,
+      [key]: value,
+    }));
+  };
 
   render() {
+    const { addMovie } = this.props;
+    const {
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    } = this.state;
+
     return (
-      <form>
-        Put the form here
+      <form className="new-movie">
+        <input
+          type="text"
+          placeholder="title"
+          name="title"
+          value={title}
+          onChange={(event) => {
+            this.handleChange(event.target.value, event.target.name);
+          }}
```

Comment:
destructure target and name from event object inside handleChange function

```suggestion
          onChange={this.handleChange}
```